doc/api/modules.markdown

```diff
@@ -429,6 +429,18 @@ of absolute paths, then io.js will search those paths for modules if they
 are not found elsewhere.  (Note: On Windows, `NODE_PATH` is delimited by
 semicolons instead of colons.)
 
+`NODE_PATH` was originally created to support loading modules from
+varying paths before the current
+[module resolution](https://iojs.org/api/modules.html) algorithm was
```

Comment:
Could possible link to https://iojs.org/api/modules.html#modules_all_together, although I don't really like this header name :)